cypress/integration/e2e/roadmap-page.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Roadmap Page', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('/page-data/roadmap/page-data.json').as('roadmapDataRequest');
+    cy.visit('/roadmap');
+    cy.wait('@roadmapDataRequest');
+
+    return cy
+      .get<{response: {body: {result: {data: {BPM: Record<string, any>[]; Roadmap: Record<string, any>[]}}}}}>(
```

Comment:
this type is super gross... going to move it to a central `types.ts` file.

Reply:
Can just define it at the start of this file until we use it more widely.